pandas/tests/test_common.py

```diff
@@ -222,3 +224,22 @@ def test_standardize_mapping():
 
     dd = collections.defaultdict(list)
     assert isinstance(com.standardize_mapping(dd), partial)
+
+
+@pytest.mark.parametrize('frame', [
+    pd.concat(100 * [DataFrame([[0.123456, 0.234567, 0.567567],
```

Comment:
Instead of using concat just multiply your list of lists by 100 within the constructor - will be more performant and idiomatic

Reply:
good point.